### ISSUE TITLE
Crash happen when waiting for receiver to connect

### ISSUE THREAD

**Issue by lakshyagupta21** [opendatakit/skunkworks-crow#297]
#### Software and hardware versions 
Android v7.1 Xiaomi Redmi 4

#### Problem description
While waiting for the receiver to connect to sender's BlueTooth, after some duration I see my app crashed. Was able to reproduce while testing #270 

#### Steps to reproduce the problem
1. Send forms using BlueTooth
2. Wait for receiver to connect.
3. Press back and wait for 120sec to see your app restart.

#### Expected behavior
App should not crash
#### Other information
Stack Trace
```
019-08-11 14:44:37.697 2753-2753/org.odk.share E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.odk.share, PID: 2753
    android.view.WindowManager$BadTokenException: Unable to add window -- token android.os.BinderProxy@59b7603 is not valid; is your activity running?
        at android.view.ViewRootImpl.setView(ViewRootImpl.java:702)
        at android.view.WindowManagerGlobal.addView(WindowManagerGlobal.java:344)
        at android.view.WindowManagerImpl.addView(WindowManagerImpl.java:94)
        at android.app.Dialog.show(Dialog.java:329)
        at org.odk.share.views.ui.bluetooth.BtSenderActivity$1.onFinish(BtSenderActivity.java:248)
        at android.os.CountDownTimer$1.handleMessage(CountDownTimer.java:127)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:163)
        at android.app.ActivityThread.main(ActivityThread.java:6238)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:933)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:823)
```

**Comment by huangyz0918** [comment]
This issue was resolved in PR #270

**Comment by huangyz0918** [comment]
@opendatakit-bot claim

**Comment by huangyz0918** [comment]
Will fix this in #270
